### ISSUE TITLE
`make witti` should do INF screen logging + JSON DBG to data directory

### ISSUE THREAD

**Issue by arnetheduck** [status-im/nim-beacon-chain#1116]
`INF` is the end-user level that we want to recommend for "normal" production use eventually, but we still need to include `DBG` by default for error reporting.

Thus, writing DBG log file to `dataDir` while displaying `INF` lets us dogfood the setup without losing important logs for bug reports.

This should apply to `make witti` and similar user-oriented targets.

**Comment by stefantalpalaru** [comment]
We would need a new feature in Chronicles for that - different log levels for different sinks.

**Comment by arnetheduck** [comment]
ok, so one step forwards on this would be to write the json log file in the data folder of the node, instead of `$PWD`

**Comment by mratsim** [comment]
This was done for Altona/Medalla

**Comment by arnetheduck** [comment]
nope, not yet - per-sink log levels not in chronicles yet (ping @zah )

**Comment by mratsim** [comment]
Mmmh, I don't remember closing this one yesterday, I think I meant to close another one sorry

**Comment by arnetheduck** [comment]
Closing, as writing log files unannounced may quickly fill up disks - the desired effect can be achieved with `tee` and `grep`
